tools/generators/epic-generator/index.spec.ts

```diff
@@ -0,0 +1,275 @@
+      const tree = createTreeWithEmptyWorkspace();
+
+      expect(() => epicGenerator(tree, { title: 'test title' })).toThrow('You are not logged into GitHub CLI (gh).');
+    });
+  });
+
+  describe('issue generation', () => {
+    it('uses defaults', () => {
+      const tree = setupTest([]);
+
+      const effectsCall = epicGenerator(tree, {
+        title: 'test title',
+      });
+      effectsCall();
+
+      expect(execSyncMock).nthCalledWith(
+        2,
+        'gh issue create --repo "microsoft/fluentui" --title "test title" --body "*Description to be added*"',
+      );
+    });
+
+    it('handles a real life scenario', () => {
+      const packages: Package[] = [
+        {
+          name: 'public-docsite',
+          version: '9.0.0',
+          projectType: 'application',
+          owners: ['@microsoft/fluentui-v8-website'],
+        },
+        {
+          name: 'react-link',
+          version: '9.0.0',
+          projectType: 'library',
+          owners: ['@microsoft/cxe-red', '@khmakoto', '@microsoft/cxe-coastal'],
+        },
+        {
+          name: 'react-card',
+          version: '9.0.0',
+          projectType: 'library',
+          owners: ['@microsoft/cxe-prg'],
+        },
+        {
+          name: 'react',
+          version: '8.0.0',
+          projectType: 'library',
+          owners: ['@microsoft/cxe-red'],
+        },
+        {
+          name: 'react-menu',
+          version: '9.0.0',
+          projectType: 'library',
+          owners: ['@microsoft/teams-prg'],
+        },
+        {
+          name: 'react-button',
+          version: '9.0.0',
+          projectType: 'library',
+          owners: ['@microsoft/cxe-red', '@khmakoto'],
+        },
+        {
+          name: 'misterious-unowned-package',
+          version: '9.0.0',
+          projectType: 'library',
+          owners: [],
+        },
+        {
+          name: 'react-slider',
+          version: '9.0.0',
+          projectType: 'library',
+          owners: ['@microsoft/cxe-coastal', '@micahgodbolt'],
+        },
+        {
+          name: 'vr-tests',
+          version: '9.0.0',
+          projectType: 'application',
+          owners: ['@microsoft/fluentui-react'],
+        },
+        {
+          name: 'react-accordion',
+          version: '9.0.0',
+          projectType: 'library',
+          owners: ['@microsoft/teams-prg', '@microsoft/cxe-coastal'],
+        },
+      ];
+      const tree = setupTest(packages);
+
+      const effectsCall = epicGenerator(tree, {
+        title: 'test title',
+        repository: 'cool-company/repository',
+      });
+      effectsCall();
+
+      // @microsoft/cxe-red issue creation
+      expect(execSyncMock).nthCalledWith(
+        3,
+        stripIndents`gh issue create --repo "cool-company/repository" --title "test title - @microsoft/cxe-red" --body "🚧 This is an auto-generated issue to individually track migration progress.
```

Comment:
all of these are lint errors 

consider `// eslint-disable-next-line @fluentui/max-len`

IMO I feel this rule is not helping at all, so I'd suggest to turn it off for whole `tools/` thoughts?